integration/framework/utils.py

```
@@ -68,7 +69,7 @@ def validate_and_sanitize_name(name: str) -> str:
     return sanitized
 
 
-def get_machine_info():
+def get_machine_info() -> str:
```

again, i really don't like the silently ignoring exception in this function. The tree
pattern where you have things like that return "Windows" is also a sign of bad code,
reminds me of callback hell.